portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,30 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  static final String CONTENT_TYPE = "application/json;";
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    List messages = new ArrayList<String>();
```

Comment:
Please correct use of type inference.

See
https://docs.oracle.com/javase/tutorial/java/generics/genTypeInference.html#:~:text=Type%20Inference,that%20make%20the%20invocation%20applicable.&text=Finally%2C%20the%20inference%20algorithm%20tries,with%20all%20of%20the%20arguments.